ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -368,6 +368,15 @@ VAV_SD_DSP_HWDC_DSPZDHC:
   - HWDC
   - DSPZDHC
 
+VAV_VSZ:
```

Comment:
Will need to be changed if the abstract type can't be justified.

Reply:
Type was removed. VSZ was removed also.